src/redux/actions/componentConfig/loadPlotConfig.js

```diff
@@ -14,15 +14,18 @@ const loadPlotConfig = (experimentId, plotUuid, plotType) => async (dispatch) =>
       const data = await response.json();
       dispatch({
         type: LOAD_CONFIG,
-        payload: data,
+        payload: {
+          ...data,
+          config: _.merge(initialPlotConfigStates[plotType], data.config),
```

Comment:
Merge `initialPlotConfig` with config stored in database. Useful to handle first loads where data from database have empty config.